src/components/modules/banners/contributions/ContributionsBanner.tsx

```diff
@@ -1,165 +1,169 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
 import React from 'react';
-import { styles } from './ContributionsBannerStyles';
-import { SvgRoundel } from '@guardian/src-brand';
-import { SvgCross, SvgArrowRightStraight } from '@guardian/src-icons';
-import { ThemeProvider } from 'emotion-theming';
-import { Button, LinkButton, buttonReaderRevenueBrandAlt } from '@guardian/src-button';
-import { Hide } from '@guardian/src-layout';
-import contributionsBannerWrapper, {
-    ContributionsBannerProps,
-    ContributionsBannerRenderedContent,
-} from './ContributionsBannerWrapper';
+import contributionsBannerWrapper, { ContributionsBannerProps } from './ContributionsBannerWrapper';
```

Comment:
The diff here is not helpful here. I've just replaced the contents of this page with what was in `ContributionsBannerVariantA.tsx`